client/src/components/MapView/LayerControl.tsx

```diff
@@ -1,6 +1,6 @@
 import type { ControlPosition } from 'react-map-gl';
 import { useControl } from 'react-map-gl';
-import { IControl, Map } from 'maplibre-gl';
+import { IControl, Map as WebMap } from 'maplibre-gl';
```

Comment:
I see why you need to do this after scanning the code and finding the other "Map", but maybe "MaplibreMap" is a better choice.